build.py

```diff
@@ -214,13 +295,41 @@ def _validate_android_args(args: argparse.Namespace):
                 )
 
         if args.cmake_generator != original_cmake_generator:
-            log.info(f"Setting CMake generator to '{args.cmake_generator}' for cross-compiling for Android.")
+            log.info(
+                f"Setting CMake generator to '{args.cmake_generator}' for cross-compiling for Android."
+            )
 
         # no C# on Android so automatically skip
         if not args.skip_csharp:
             args.skip_csharp = True
 
 
+def _validate_ios_args(args: argparse.Namespace):
+    if args.ios:
+        # TODO: cross compile for iOS on Linux?
```

Comment:
Is cross compiling on Linux possible?

Reply:
right - I think it's possible but there are probably a couple issues as cross build ios on Linux, such as requires a complicated tool chain setup/no easier build set up for ios simulator for CI testing purposes, etc..

removed this comment. just put it there as a question/potential todo.